### PR TITLE
Issue #11163: Enforce file size on all InputMagicNumber

### PR DESCRIPTION
**Pull request by sd9616**
Issue #11163: Enforces file size on all InputMagicNumber
Split each MagicNumber file into 3 files each 120 lines or less. 
Updated MagicNumberCheckTest accordingly. 
Ran mvn clean verify successfully
Ran inspections successfully

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreHashCodeMethod1.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+class ArrayMagicTestIgnoreHashCodeMethod1
+{
+    private static final int[] NONMAGIC = {3};
+    private int[] magic = {3}; // violation
+    private static final int[][] NONMAGIC2 = {{1}, {2}, {3}};
+}
+
+
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreHashCodeMethod1.java:103]: Remove extra empty line at the end of file.

Here in bunch of other files

**Review by romani — CHANGES_REQUESTED** [review]
Item to improve

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreHashCodeMethod1.java (hunk 1 above)
Comment (sd9616) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreHashCodeMethod1.java:103]: Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreFieldDeclaration1.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  int HIGH = 78;
+}
+
+class ArrayMagicTestIgnoreFieldDeclaration1
+{
+    private static final int[] NONMAGIC = {3};
+    private int[] magic = {3};
+    private static final int[][] NONMAGIC2 = {{1}, {2}, {3}};
+}
+
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIgnoreFieldDeclaration1.java:102]: Please Remove extra empty lines in all files you introduce

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Review by romani — APPROVED** [review]
Ok to merge if CI pass

**Comment by sd9616** [comment]
I'm not sure how my changes caused these errors since the changes just removed empty lines.

**Comment by romani** [comment]
looks like problems to downlaod groovy , infrastricture problems, but we need pitest to pass on your changes.
I will rerun them later.

**Comment by nrmancuso** [comment]
@sd9616 I have rebased your PR branch, let's see if we have success this time

**Comment by romani** [comment]
Semaphore failure is not releated.

**Pull request merged**